### PR TITLE
added the text views for displaying scoreSum, numberOfCodes. Currentl…

### PR DESCRIPTION
**Pull request by real-rookie**
[#7](https://github.com/CMPUT301W23T40/CaptureTheQR/issues/7), [#8](https://github.com/CMPUT301W23T40/CaptureTheQR/issues/8)

### CODE REVIEW

**Comment by rachel-ellis** [comment]
could you match this design more? Otherwise lmk and I can fix it
Screenshot 2023-03-27 at 11 04 09 AM

**Comment by real-rookie** [comment]
> could you match this design more? Otherwise lmk and I can fix it
> Screenshot 2023-03-27 at 11 04 09 AM

Sorry I am not sure how to align the numbers to the right… It would be great if you can fix it. Thanks

**Comment by rachel-ellis** [comment]
> > could you match this design more? Otherwise lmk and I can fix it
> > Screenshot 2023-03-27 at 11 04 09 AM
> 
> Sorry I am not sure how to align the numbers to the right… It would be great if you can fix it. Thanks

should be good now
Screenshot 2023-03-27 at 3 32 48 PM

**Review by rachel-ellis — DISMISSED** [review]
lgtkm!!! not sure if the toast is necessary (could be distracting), but also makes it obv which metric is used.

looks like the rank at the bottom works now (yay!!), so only frontend improvements and showing the correct metric in the list (last value), which is because it's hardcoded to always show the highScore field instead of the current metric in ScoreboardList (I can fix that with UI changes)

also, if we aren't using the rank in the DB can we clean that up @real-rookie

**Comment by rachel-ellis** [comment]
> lgtkm!!! not sure if the toast is necessary (could be distracting), but also makes it obv which metric is used.
> 
> looks like the rank at the bottom works now (yay!!), so only frontend improvements and showing the correct metric in the list (last value), which is because it's hardcoded to always show the highScore field instead of the current metric in ScoreboardList (I can fix that with UI changes)
> 
> also, if we aren't using the rank in the DB can we clean that up @real-rookie

sorry ignore, this was for the other PR lol

File: app/src/main/java/com/cmput301w23t40/capturetheqr/OtherPlayerActivity.java [hunk 1]
```diff
@@ -36,6 +39,9 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onCallBack(QRCode maxQR, QRCode minQR) {
                 if(maxQR != null) {
+                    // FIXME check if the following two lines work after merging Anushka's PR
```
Comment (ekhumbata) [inline app/src/main/java/com/cmput301w23t40/capturetheqr/OtherPlayerActivity.java]: I don't see a pr anymore for Anushka? Do these work? if so please remove the comment

**Review by ekhumbata — APPROVED** [review]
lgtm

**Pull request merged**